### PR TITLE
kernel: Enable selinux support.

### PR DESCRIPTION
**Pull request by mkosola**
[kernel] Enable selinux support. JB#43487

Signed-off-by: Matti Kosola <matti.kosola@jolla.com>

### CODE REVIEW

File: arch/arm64/configs/aosp_loire_suzu_defconfig [hunk 1]
```diff
@@ -708,3 +707,34 @@ CONFIG_LOGCAT_SIZE=256
...
+CONFIG_LSM_MMAP_MIN_ADDR=65536
+CONFIG_SECURITY_SELINUX=y
+CONFIG_SECURITY_SELINUX_BOOTPARAM=y
+CONFIG_SECURITY_SELINUX_BOOTPARAM_VALUE=1
+# CONFIG_SECURITY_SELINUX_DISABLE is not set
+CONFIG_SECURITY_SELINUX_DEVELOP=y
+CONFIG_SECURITY_SELINUX_AVC_STATS=y
+CONFIG_SECURITY_SELINUX_CHECKREQPROT_VALUE=1
+# CONFIG_SECURITY_SELINUX_POLICYDB_VERSION_MAX is not set
+CONFIG_SECURITY_APPARMOR=y
```
Comment (LaakkonenJussi) [inline arch/arm64/configs/aosp_loire_suzu_defconfig]: Out of curiosity, why apparmor is enabled? AFAIK it is not needed with SELinux?

Comment (mkosola) [reply arch/arm64/configs/aosp_loire_suzu_defconfig]: It was needed to be enabled before you were able to enable SELinux from menuconfig

Comment (mkosola) [reply arch/arm64/configs/aosp_loire_suzu_defconfig]: Ah, that is not needed. My bad, it was audit which was not enabled and that is why selinux was missing from menuconfig, will fix.

Comment (LaakkonenJussi) [reply arch/arm64/configs/aosp_loire_suzu_defconfig]: Right. Without audit in config SELinux did not get enabled even with manually built kernel. Not sure why, but it apparently has to be always enabled. :)

File: arch/arm64/configs/aosp_loire_suzu_defconfig [hunk 2]
```diff
@@ -708,3 +707,25 @@ CONFIG_LOGCAT_SIZE=256
...
+CONFIG_NETWORK_SECMARK=y
+# CONFIG_NF_CONNTRACK_SECMARK is not set
+# CONFIG_NETFILTER_XT_TARGET_AUDIT is not set
+# CONFIG_NETFILTER_XT_TARGET_SECMARK is not set
+CONFIG_LSM_MMAP_MIN_ADDR=65536
+CONFIG_SECURITY_SELINUX=y
+CONFIG_SECURITY_SELINUX_BOOTPARAM=y
+CONFIG_SECURITY_SELINUX_BOOTPARAM_VALUE=1
+# CONFIG_SECURITY_SELINUX_DISABLE is not set
+CONFIG_SECURITY_SELINUX_DEVELOP=y
```
Comment (jusa) [inline arch/arm64/configs/aosp_loire_suzu_defconfig:723]: this could be N

**Review by jusa — APPROVED** [review]

**Pull request merged**